hbase-handler/src/java/org/apache/hadoop/hive/hbase/ResultWritable.java

```diff
@@ -26,8 +26,8 @@
 import org.apache.hadoop.hbase.KeyValue;
 import org.apache.hadoop.hbase.KeyValueUtil;
 import org.apache.hadoop.hbase.client.Result;
-import org.apache.hadoop.hbase.protobuf.ProtobufUtil;
-import org.apache.hadoop.hbase.protobuf.generated.ClientProtos;
+import org.apache.hadoop.hbase.shaded.protobuf.ProtobufUtil;
+import org.apache.hadoop.hbase.shaded.protobuf.generated.ClientProtos;
```

Comment:
Do we know why these imports changed and point to shaded classes?

Reply:
The Hbase tests were failing because they had one protobuf 2.5.0 reference for some use case. So bugged the HBase folks and got that as a solution.
https://issues.apache.org/jira/browse/HBASE-21587?focusedCommentId=17575116&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17575116

Reply:
OK, let me rephrase a bit to be sure I understand the current situation. After the changes in this PR, most parts of Hive will use protobuf 3.21.4 while the HBase module will keep using protobuf 2.5.0 via the shaded packages. Is that correct?

Reply:
Yes